rules/python_rules.build_defs

```diff
@@ -129,9 +216,8 @@ def python_binary(name:str, main:str, srcs:list=[], resources:list=[], out:str=N
                      determine something appropriate for the given interpreter.
       labels (list): Labels to apply to this rule.
     """
-    shebang = shebang or interpreter or CONFIG.DEFAULT_PYTHON_INTERPRETER
-    cmd = '$TOOLS_PEX -s "%s" -m "%s" --zip_safe --interpreter_options="%s" --stamp="$STAMP"' % (
-        shebang, CONFIG.PYTHON_MODULE_DIR, CONFIG.PYTHON_INTERPRETER_OPTIONS)
+    cmd = '$TOOLS_PEX -s "$TOOLS_INTERPRETER" -m "%s" --zip_safe --interpreter_options="%s" --stamp="$STAMP"' % (
```

Comment:
This command create a pex file with the shebang like this:
```
#!/home/tom/dev/tcn/m/neo/plz-out/bin/tools/python/python/bin/python -S
```
IMO we should have a shebang like this: 
```
#!/usr/bin/env /home/tom/dev/tcn/m/neo/plz-out/bin/tools/python/python/bin/python -S || /usr/bin/env python3 -S
```
to ensure the resulted pex is portable. No matter how I tried I wasn't able to generate that.
If you are interested in that I could use a hand here.